packages/platform/platform-fastify/test/test-auth.integration.spec.ts

```
@@ -0,0 +1,345 @@
+import {Middleware, UseAuth} from "@tsed/platform-middlewares";
+import {Context} from "@tsed/platform-params";
+import {Get, In, Post, Returns, Security} from "@tsed/schema";
+import SuperTest from "supertest";
+import {afterAll, beforeAll, describe, expect, it} from "vitest";
+
+import baseSpec from "../data/swagger.json";
+import {PlatformTestingSdkOpts} from "../interfaces/index.js";
+
+@Injectable()
+export class TokenService {
+  private _token: string = "EMPTY";
```

Biome (1.9.4)

[error] This type annotation is trivially inferred from its initialization.

Safe fix: Remove the type annotation.


(lint/style/noInferrableTypes)